src/index.js

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+import ReactDOM from "react-dom/client";
+import { Provider } from "react-redux";
+import store from "./redux/store";
+import App from "./App";
+
+const root = ReactDOM.createRoot(document.getElementById("root"));
+root.render(
+  <React.StrictMode>
+    <Provider store={store}>
+      <App />
+    </Provider>
+  </React.StrictMode>
```

Comment:
- [ ] Dear Team, Please keep in mind for this milestone you are expected to write unit tests with React Testing Library and Jest (for unit tests). Testing is one major part of this day's development and provides an assertion over your overall project. Please add tests using the `React Testing Library and Jest`